pkg/logs/internal/decoder/auto_multiline_detection/json_detector.go

```diff
@@ -8,7 +8,7 @@ package automultilinedetection
 
 import "regexp"
 
-var jsonRegexp = regexp.MustCompile(`^\s*\{\s*\"`)
+var jsonRegexp = regexp.MustCompile(`^^\s*\{\s*(\"|})`)
```

Comment:
Duplicate `^`?

Also, may be a smidge simpler with `["}]`:

```suggestion
var jsonRegexp = regexp.MustCompile(`^\s*\{\s*["}]`)
```

Reply:
🤦 . Copy paste error :( 
Surprised the go regex engine was happy with this. 
Also good idea - will do.